GiniSwitchSDK/GiniSwitchSDK/Classes/PagesCollectionViewController.swift

```diff
@@ -63,8 +68,8 @@ class PagesCollectionViewController: UIViewController {
     fileprivate func setupOptionsButton() {
         // change the color of the button's image to make it reflect the
         // app's theme
-        guard let buttonImage = optionsButton.image(for: .normal) else {
-            logger.logError(message: "Options button didn't have an image")
```

Comment:
You don't need to log this error again, right?

Reply:
No, since I'm also calling this in the `didSet` for the `theme` property it's more likely that this guard triggers. And it wouldn't be an error - the button will just get setup after the view loads.

So short answer, it's no longer exceptional behaviour.